it/support/IntegrationBaseSpec.scala

```diff
@@ -0,0 +1,12 @@
+package support
+
+import play.api.libs.ws.WS
+import uk.gov.hmrc.play.it.ServiceSpec
+import play.api.Play.current
+
+class IntegrationBaseSpec(testName: String, extraConfig: Map[String, String] = Map.empty) extends ServiceSpec{
+  private val testNameMaxLength = 30
```

Comment:
this will hide problems, maybe we should not `takeRight` and just let it blow up